versions/latest/modules/en/pages/rancher-admin/scc-registration.adoc

```diff
@@ -0,0 +1,105 @@
+= SUSE Customer Center Registration
+:revdate: 2025-08-19
+:page-revdate: {revdate}
+
+Since Rancher v2.12.1, you should now register your SUSE Rancher Manager Prime Suite with https://scc.suse.com/home[SUSE Customer Center (SCC)] from the Rancher UI. This will enable you to see your registered systems in SCC under your subscription.
+
+== Prequsities
+
+Registering your Rancher Manager instance with SUSE Customer Center requires a SUSE account. If you don't have a SUSE account yet, visit the https://scc.suse.com/[SCC home page] to create one.
+
+You must also have access to your organization. To join your organization, you may either:
+
+* Ask your organization's administrator to add you to the account. See our https://scc.suse.com/docs/userguide#UG-Requesting-Access-to-an-Organizations-Account[documentation] for details.
+* If you know one of your organization's registration codes, enter it when prompted.
+
+== Register with SUSE Customer Center
+
+Check the Notification Center (bell icon) in the upper-right corner of your Rancher dashboard for a "Rancher Prime Registration" notification and click *Register Now*. This redirects you to the *Registration* page under *Global Settings*. 
+
+You have the option to register your Rancher Manager instance with SCC <<online,Online Registration>> or <<offline,Offline Registration>> from the Rancher UI.
+
+=== Online Registration
+
+You can register online through the Rancher UI or with Helm.
+
+==== Rancher UI
+
+. Get the Registration code for your Rancher Manager subscription on SCC. Log in to SCC, navigate to **My Organizations** and click on the organization with the Rancher Manager subscription. 
+. Find your Registration code under *Organization* > *Subscriptions* > *Subscription Information*. 
```

Comment:
These steps to get a registration code should be part of the prerequisites since it's needed regardless of the method used to register.

There's also some inconsistency right now because these steps are only part of the UI section, but not part of the Helm section.